fmfexporter/fmf_testcase.py

```diff
@@ -153,6 +160,11 @@ def get_fmf_data(node, data, _default):
             ts = get_fmf_data(fmf_node, 'testsuite', {})
             fmf_tc.testsuite = FMFTestSuite(**ts)
 
+            # Test case automation script
+            fmf_tc.automation_script = get_fmf_data(fmf_node, 'automation_script', '')
```

Comment:
If we need an "automation_script" in the FMF TC, then we must first modify the Schema.
Do you think it might be a generic Test case element or just something internal to Polarion element of the FMF Metadata?

Reply:
Yes, generic thing per test suite should be enough imo. (How to execute it).
The only 'non-generic' thing in my opinion in this, is addition of test-class/test-name into automation script. (Or, as I added - "see test steps :) )